# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/firefox/hotfix-icuuc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- firefox-136.0/js/moz.configure.vanilla	2025-03-04 15:46:22.196971064 +0100
+++ firefox-136.0/js/moz.configure	2025-03-04 15:46:30.868792629 +0100
@@ -1250,7 +1250,7 @@
     return enable_system_icu
 
 
-system_icu = pkg_check_modules("MOZ_ICU", "icu-i18n >= 76.1", when="--with-system-icu")
+system_icu = pkg_check_modules("MOZ_ICU", "icu-i18n >= 76.1, icu-uc >= 76.1", when="--with-system-icu")
 
 
 @depends(enable_system_icu_option)
